src/components/BorrowerProfile/SupporterDetails.vue

```diff
@@ -150,7 +150,6 @@ export default {
 			userCardStyleOptions: [
 				{ color: 'tw-text-action', bg: 'tw-bg-brand-50' },
 				{ color: 'tw-text-black', bg: 'tw-bg-brand-100' },
-				{ color: 'tw-text-action', bg: 'tw-bg-secondary' },
```

Comment:
I thought this fix was needed in the new homepage BP? Not in the old BP.

Reply:
This is for old page, for new page we are using KvUserAvatar, https://main--608b4cf87f686c00213841b1.chromatic.com/?path=/docs/kvuseravatar--kv-components

Reply:
New page is not using a BG similar than KvUserAvatar...

Reply:
Ah gotcha, so this is an old issue, I had assumed it was just for the new page, thanks for the info.